app/controllers/api/cloud_networks_controller.rb

```diff
@@ -9,5 +9,17 @@ def delete_resource(type, resource_id, _data = {})
       task_id = cloud_network.delete_cloud_network_queue(User.current_user.userid)
       action_result(true, "Deleting #{cloud_network.name}", :task_id => task_id)
     end
+
+    def options
+      return super unless %w[id ems_id].include_any?(params.keys)
+
+      if params.key?("ems_id")
+        ems = ExtManagementSystem.find(params["ems_id"])
+        render_options(:cloud_networks, :form_schema => CloudNetwork.class_by_ems(ems).params_for_create(ems))
+      else
+        cloud_network = CloudNetwork.find(params["id"])
+        render_options(:cloud_networks, :form_schema => cloud_network.params_for_edit)
+      end
+    end
```

Comment:
NOTE: this is an example of how class and instance DDF params can be returned by calling `OPTIONS /api/cloud_networks` and `OPTIONS /api/cloud_networks/:id`